ITEM.tsv

```diff
@@ -3472,8 +3472,8 @@ ITEM_20150317_003471	Scot's First Report
 ITEM_20150317_003472	{memo X}A report of the forgotten Naru's development records. 
 ITEM_20150317_003473	Supply Pouch 
 ITEM_20150317_003474	A pouch of supplies looted by Vubbes. Let's take these to Yukurt to investigate the odd behaviour of the Vubbes. 
-ITEM_20150317_003475	Doki Oil 
-ITEM_20150317_003476	Oil that reeks with a Doki's scent. 
+ITEM_20150317_003475	$Duckey Fat 
```

Comment:
Yep, oil is possible. But most people don't say chicken oil, they say chicken fat/ duck fat. 

It's just my preference because when I hear oil, I imagine a liquid substance, whereas this item seems to be a solid substance